pkgs/data/misc/geolite-legacy/default.nix

```diff
@@ -1,27 +1,28 @@
-{ lib, stdenv, fetchurl, zstd }:
+{ lib, stdenv, fetchzip, zstd }:
```

Comment:
I notice it wasn't formatted according to nixfmt-rfc-style yet before.

But can you add a second commit where you run nixfmt on the file?